modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/BaseProjectTemplatesTestCase.java

```diff
@@ -1302,8 +1302,8 @@ public default List<String> sanitizeLines(List<String> lines) {
 	public default void testBuildTemplateNpm(
 			TemporaryFolder temporaryFolder, MavenExecutor mavenExecutor,
 			String template, String name, String packageName, String className,
-			String liferayVersion, String nodePackageManager, String product,
-			URI gradleDistribution)
+			String liferayVersion, String nodePackageManager,
+			String liferayProduct, URI gradleDistribution)
```

Comment:
Move liferayProduct before liferayVersion everywhere (match pattern in archetype-metadata.xml)